bin/i18n-codeorg/in.sh

```diff
@@ -20,7 +20,6 @@ mkdir -p $loc_dir
 cp_in $orig_dir/en.yml $loc_dir/base.yml
 
 # Copy in needed files from dashboard
-cp_in $orig_dir/blocks.en.yml $loc_dir/blocks.yml
```

Comment:
Why did we copy this file, if we edited the original file later in [this line](https://github.com/code-dot-org/code-dot-org/pull/52843/files#diff-9b74aafab3e6e0ca501ff82f47a8c610037c21ec1cb0960c0a823055e3cd68d1R41:~:text=.write(-,dashboard_i18n_file_path,-%2C%20I18nScriptUtils.)

Reply:
We basically create the original i18n file `dashboard/config/locales/blocks.en.yml` first and then copy it to the i18n source folder to upload it to Crowdin

Reply:
I'm glad we are centralizing components of one feature that were distributed in multiple files/scripts.